README.md

```diff
@@ -233,6 +233,47 @@ func Example() {
 }
 ```
 
+
+## Broadcasting Messages
+Broadcasting messages is a way for the client to send messages to a group of actors. There
+are currently two different strategies for message broadcasting:
+
+ - First-one-wins, where the request context is canceled as soon as one actor responds to the message.
+ - Guaranteed delivery to all actors
```

Comment:
Can we change bullet number two to read:

 - Delivery to all actors, waits for all responses or timeouts.

I don't think we should describe it as guaranteed, since even with infinite retries it would not be that. Sorry to be pedantic about it.